config/api.yml

```diff
@@ -1124,10 +1128,6 @@
       :get:
       - :name: show
       - :identifier: miq_cloud_networks_view
-    :load_balancers_subcollection_actions:
-      :get:
-      - :name: show
-        :identifier: load_balancer_show
```

Comment:
why moving these out of here ?  keeping them here gives us finer granularity on sub*/roles based on collection.